docs/data/toolpad/concepts/deployment.md

```diff
@@ -14,30 +16,64 @@ Make sure to add the following scripts to your `package.json`
  }
 ```
 
-## Build steps
+## Install step
+
+Install needed depdencies via:
+
+<codeblock storageKey="toolpad-deploy-package-manager">
+
+```bash npm
+npm install
+```
+
+```bash yarn
+yarn
+```
+
+</codeblock>
+
+## Build step
 
 This command will create an optimized production build for the Toolpad app and will output the generated files:
 
-```bash
+<codeblock storageKey="toolpad-deploy-package-manager">
```

Comment:
The storage key is to remember the setting across reloads? Should we use one single key for the whole of the Toolpad site?

Reply:
Yes, makes more sense